### ISSUE TITLE
Report an error in Typescript

### ISSUE THREAD

**Issue by jasonsoft-net** [datalust/bunyan-seq#34]
```
const log = bunyan.createLogger({
  name: 'card-service',
  streams: [
    {
      stream: process.stdout,
      level: 'warn',
    },
    seq.createStream({
      serverUrl: 'http://localhost:5341',
      level: 'info',
    }),
  ],
});
```

<img width="936" alt="WX20210629-194451@2x" src="https://user-images.githubusercontent.com/22167571/123791902-96593700-d912-11eb-80d3-3a4ebd26c97a.png">

**Comment by nblumhardt** [comment]
Thanks Jason - looks like `createStream()` should be returning a `Stream` in https://github.com/datalust/bunyan-seq/blob/dev/index.d.ts ? (Typings at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bunyan/index.d.ts#L224)

**Comment by nblumhardt** [comment]
This should be fixed in 0.8.0 - any better luck with that version? Thanks Jason!
